src/main/java/carrace/domain/Car.java

```diff
@@ -1,12 +1,33 @@
 package carrace.domain;
 
+import java.util.Objects;
+
 public class Car {
 
+    private static final int DEFAULT_POSITION = 1;
+    private static final int MOVE_AMOUNT = 1;
+
     private final String name;
-    private int position = 1;
+    private final int position;
 
     public Car(String name) {
+        this(name, DEFAULT_POSITION);
+    }
+
+    private Car(String name, int position) {
         this.name = name;
+        this.position = position;
+        validateName();
```

Comment:
```suggestion
        validateName(name, position);
        this.name = name;
        this.position = position;
```

빠른 반환을 통해서 유효성 검증하는 것도 좋은 방법 입니다 :)
조금이지만 변수에 할당되는 메모리의 이점이 있습니다